src/RpcServer/RpcServer/config.json

```diff
@@ -10,6 +10,7 @@
     "MaxGasInvoke": 20,
     "MaxFee": 0.1,
     "MaxConcurrentConnections": 40,
-    "DisabledMethods": [ "openwallet" ]
+    "DisabledMethods": [ "openwallet" ],
+    "Network": 5195086
```

Comment:
We can run `RpcServer`s of different networks on different ports.

Reply:
Do you mean the `port` attribute in config file?